src/utils/utils-contour.js

```diff
@@ -42,11 +45,14 @@ export const buildContourSQL = (state, mapBounds, isPolygons = false) => {
     flip_latitude = false,
     contour_value_field = "z",
     lat_field = "raster_lat",
-    lon_field = "raster_lon"
+    lon_field = "raster_lon",
+    interval
   } = data
 
+  const { intervalTypeFieldName = "is_major", minor } = interval
```

Comment:
i know we have assurances from the immerse side that the major interval is divisible by minor, but I can't help but wonder if we should validate that still

Reply:
I'll add a little "validateContourState" utility to check for things

Reply:
Added some validation of the state passed in for contours! Also didn't see anything obvious but is there another/ better way than just throwing a new Error() in charting? Looks like those would bubble up and get caught in immerse sagas/thunks and the chart would display the error, so should work.